data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,495 @@ mission "Unfettered returning home"
+			# This is a "warning" for the incoming extra ship that happens because you killed/captured 10 Shield Beetles or so.
+			`	He suddenly look at you intensively, straight in the eyes, "We sometimes bend the rules for our friends, but that is not your case," he says with a sudden change of tone, before leaving.`
+				to display
+					"reputation: Hai (Unfettered)" < -2000
+			`	Mera comes back, looking a bit impatient. "Are you going to talk all day, or are you going to fight?" The Unfettered pilots immediately head to their ships. The engineer pulls you in close once they are far enough away. "I want this done by the end of the day, so do not try my patience. You are lucky I offered this deal in the first place, they were planning to take the relic by force. Now, quick, get to your ship!" You see the contestants have already begun the launching procedures and launch as soon as you can.`
+			`	(Remember to park your other ships and make sure your flagship is the Anomalocaris.)`
+				accept
+			label headingout
+			action
+				"reputation: Hai (Unfettered)" = "old reputation: Hai (Unfettered)"
+				set "unfettered: fled with the relic"
+			`	Seeing as they are unlikely to cooperate you quickly head to your ship and launch before they have time to react, but it seems like you are being followed.`
+				launch
+	on enter
+		conversation
+			branch fleeing
+				has "unfettered: fled with the relic"
+			`As you leave the planet's atmosphere, you see the challengers already waiting for you in the system. They hail you, "Luck has nothing to do with it. Show us what you got, monkey! But don't go all out either, for we can't learn if we're dead."`
+				accept
+			label fleeing
+			`The Unfettered don't seem too pleased to see you leaving with their prize. The challengers are already waiting for you in orbit, and hail you. "Where do you think you're going, monkey?"`
+			action
+				fail
+	on disabled
+		fail
+		# This is needed because the mission isn't removed until the next time you land.
+		conversation
+			`The enemy ships hail you. "Thought you could get away from us so easily? You better bribe us and then land on our worlds quick, before we value blowing you up higher than preserving that ship!"`
+				to display
+					has "unfettered: fled with the relic"
+				decline
+			`The enemy ships hail you. "You have failed the challenge. We will take the ship from you when we see fit. I'll fix your ship and leave you alone for a bribe, but I suggest you land on an Unfettered world and return the ship to us, with a most generous compensation. If you try to keep it from us, we'll do things the hard way."`
+			`	"Be warned, however, that your truce with us ends as soon as you land. This is not necessarily the end, <last>, you will simply have to prove yourself another way."`
+			`	"Oh, and a word of advice: maybe you could have outfitted your ship differently, or used your range advantage by moving away at full thrust, before turning back to fire on us."`
```

Comment:
```suggestion
			`	"Be warned, however, that your truce with us ends as soon as you land. Today, you've failed us, but maybe someday you will prove yourself otherwise."`
```
We don't really hold the player's hand and tell them what to do. For example, we don't tell the player how to do dodge nukes in FW or dodge Archon shots in Remnant missions; that's something they figure out. There could be several ways to do this, too.